rct229/rule_engine/rule_base.py

```diff
@@ -120,6 +124,7 @@ def evaluate(self, rmrs, data=None):
                         # Determine if manual check is required
                         if self.manual_check_required(context, calc_vals, data):
                             outcome["result"] = "UNDETERMINED"
+                            outcome["message"] = self.manual_check_required_msg
```

Comment:
To reduce the size of the output, how about including the message conditionally?
```
if self.manual_check_required_msg:
    outcome["message"] = self.manual_check_required_msg
```

And the same for the NOT_APPLICABLE case.

Reply:
Updated